src/server_manager/web_app/app.ts

```diff
@@ -290,11 +290,12 @@ export class App {
 
     appRoot.addEventListener('OpenShareDialogRequested', (event: CustomEvent) => {
       const accessKey = event.detail.accessKey;
-      this.appRoot.openShareDialog(accessKey, this.getS3InviteUrl(accessKey));
+      this.appRoot.openShareDialog(accessKey);
     });
 
     appRoot.addEventListener('OpenGetConnectedDialogRequested', (event: CustomEvent) => {
-      this.appRoot.openGetConnectedDialog(this.getS3InviteUrl(event.detail.accessKey, true));
```

Comment:
We shouldn't have the get connected button take to the share dialog. 

The change in behavior should be accompanied by the actual change of the manager key UI. It should delete events and image resources. It's independent of how the share ui looks like, so let's put it in a separate PR.

Reply:
Ah, from our email exchange, I thought we just wanted to quickly point this to that location in the interest of time. Sure. I'll need designs.

Reply:
Reverted.